src/openrct2/windows/_legacy.cpp

```diff
@@ -345,13 +345,13 @@ bool window_ride_construction_update_state(
     bool turnOffLiftHill = false;
     if (!(_enabledRidePieces & (1ULL << TRACK_LIFT_HILL_CURVE)))
     {
-        if (TrackFlags[trackType] & TRACK_ELEM_FLAG_CURVE_ALLOWS_LIFT)
+        if (!!(TrackFlags[trackType] & TrackElemFlag::AllowLiftCurve))
```

Comment:
This doesn't seem necessary, considering we already have an `if` (which checks for truthy). If it is absolutely necessary, then please use `!= 0` (our convention).

Reply:
I've taken a different approach to this in the latest version of the PR